account_multicurrency_revaluation_type/tests/test_currency_revaluation.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import date
+import re
+from odoo.tests.common import SavepointCase
+
+
+class TestCurrencyRevaluationType(SavepointCase):
```

Comment:
Why don't you use https://github.com/OCA/account-closing/blob/12.0/account_multicurrency_revaluation/tests/test_currency_revaluation.py#L9 ?

As most of the code is the same, you can use this class and call `super` in `setUpClass`, then do what's needed for monthly rates.

Reply:
it was easier to found where  was mistake with revaluation, but yes we can reuse them

Reply:
Can you update to make use of this test class or at least add a TODO ?